ksqldb-parser/src/main/java/io/confluent/ksql/parser/tree/TableElement.java

```diff
@@ -128,4 +133,13 @@ public String toString() {
         + ", namespace=" + namespace
         + '}';
   }
+
+  public static Namespace getNamespaceFromContext(final SqlBaseParser.TableElementContext context) {
```

Comment:
Should this method be part of the `Namespace` enum instead? Perhaps a `Namespace.of(context)`

Reply:
oh nice thats clean. moved